.circleci/config.yml

```diff
@@ -6,18 +6,29 @@ jobs:
       - image: "cimg/elixir:1.13.3"
     steps:
       - checkout
+      - run: mix local.hex --force
+      - run: mix deps.get
       - run: mix run
   test:
     docker:
       - image: "cimg/elixir:1.13.3"
     steps:
       - checkout
+      - run: mix local.hex --force
+      - run: mix deps.get
       - run: mix test
+  credo:
+    docker:
+      - image: "cimg/elixir:1.13.3"
+    steps:
+      - checkout
+      - run: mix local.hex --force
+      - run: mix deps.get
+      - run: mix credo
 
 workflows:
   build_and_test:
     jobs:
+      - credo
```

Comment:
I'm curious how running jobs in parallel affect the quota(minutes) for the free tier account.

Reply:
Again not something I thought of 😂 thank you! From what I understand after looking at a circleci blog, the number of credits used is based on the type of compute done and the number of minutes its used. I think jobs running at the same time saves credits compared to a sequential run, so long that we don't go over the amount of jobs able to be run in parallel for the free tier.